src/main/scala/beam/analysis/RideHailUtilizationCollector.scala

```diff
@@ -51,8 +56,11 @@ class RideHailUtilizationCollector(beamSvc: BeamServices)
   private val rides: ArrayBuffer[RideInfo] = ArrayBuffer()
   private val utilizations: ArrayBuffer[Utilization] = ArrayBuffer()
   private var rideHailChoices: Int = 0
+  private var rideHailPooledChoices: Int = 0
   private var rideHailInAlternatives: Int = 0
+  private var rideHailPooledInAlternatives: Int = 0
   private var totalModeChoices: Int = 0
+  private val replanningReasonToCounter: mutable.Map[String, Int] = new mutable.HashMap[String, Int]()
```

Comment:
The name caused me some confusion until I read it with emphasis on the To. To avoid that for future users, how about a name more like `replanningReasonToTotalCountMap` or something akin to that. Maybe it's just me, so feel free to ignore.